src/main/scala/com/github/retronym/sbtxjc/SbtXjcPlugin.scala

```diff
@@ -19,16 +19,18 @@ object SbtXjcPlugin extends Plugin {
   val xjcLibs        = SettingKey[Seq[ModuleID]]("xjc-libs", "Core XJC libraries")
   val xjcPlugins     = SettingKey[Seq[ModuleID]]("xjc-plugins", "Plugins for XJC code generation")
   val xjcCommandLine = SettingKey[Seq[String]]("xjc-plugin-command-line", "Extra command line parameters to XJC. Can be used to enable a plugin.")
+  val xjcBindings    = SettingKey[Seq[String]]("xjc-plugin-bindings", "Binding files to add to XJC.")
 
   /** Main settings to enable XSD compilation */
-  val xjcSettings     = Seq[Project.Setting[_]](
+  val xjcSettings     = Seq[Def.Setting[_]](
     ivyConfigurations ++= Seq(XjcTool, XjcPlugin),
     xjcCommandLine    := Seq(),
+    xjcBindings       := Seq(),
     xjcPlugins        := Seq(),
     xjcLibs           := Seq(
-      "javax.xml.bind" % "jaxb-api" % "2.1",
-      "com.sun.xml.bind" % "jaxb-impl" % "2.1.9",
-      "com.sun.xml.bind" % "jaxb-xjc" % "2.1.9"
+      "javax.xml.bind" % "jaxb-api" % "2.2.7",
```

Comment:
I tested 2.2.11 and found they screwed up their pom file, so maybe it'd be easier to wait to upgrade until they fix that. I filed a bug for it here: https://java.net/jira/browse/JAXB-1070